### DISCUSSION TITLE
Upload file to FTP from watch folder then move out of watch folder

### DISCUSSION THREAD

**Discussion by MikeG979** [iterate-ch/cyberduck#15071] [Q&A]
Hi All,

I'm trying to set up a watch folder on my desktop, which uploads incoming files to my FTP server, and after the upload is complete, I need to automatically move the file to a different folder. Is this possible with cyberduck? Thanks!

**Comment by tatze96** [comment]
Cyberduck can't move items by itself automatically. However, you could synchronize the local watch folder with a remote one using the [synchronize feature](https://docs.cyberduck.io/cyberduck/sync/) of Cyberduck and move the items into their target directories afterward.
